docs/EVG.md

```diff
@@ -66,1377 +62,1294 @@ The CA/Browser Forum is a voluntary open organization of certification authoriti
 
-  2. Residential Address at which he/she can be located;
+2. **Government Entity Subjects**: Unless verified under subsection (6), all items listed in [Section 11.2.1](#1121-verification-requirements) (2) MUST either be verified directly with, or obtained directly from, one of the following:
+   i. a Qualified Government Information Source in the political subdivision in which such Government Entity operates;
+   ii. a superior governing Government Entity in the same political subdivision as the Applicant (e.g. a Secretary of State may verify the legal existence of a specific State Department), or
+   iii. from a judge that is an active member of the federal, state or local judiciary within that political subdivision.
 
-  3. Date of birth; and
+   Any communication from a judge SHALL be verified in the same manner as is used for verifying factual assertions that are asserted by an Attorney as set forth in [Section 11.11.1](#11111-verified-legal-opinion).
 
-  4. An affirmation that all of the information contained in the Certificate Request is true and correct.
+   Such verification MAY be by direct contact with the appropriate Government Entity in person or via mail, e-mail, Web address, or telephone, using an address or phone number obtained from a Qualified Independent Information Source.
 
-(ii) A current signed government-issued identification document that includes a photo of the Individual and is signed by the Individual such as:
+3. **Business Entity Subjects**: Unless verified under subsection (6), Items listed in [Section 11.2.1](#1121-verification-requirements) (3) (A) through (C) above, MUST be verified directly with, or obtained directly from, the Registration Agency in the Applicant's Jurisdiction of Registration. Such verification MAY be performed by means of a Qualified Government Information Source, a Qualified Governmental Tax Information Source, or by direct contact with the Registration Agency in person or via mail, e-mail, Web address, or telephone, using an address or phone number obtained directly from the Qualified Government Information Source, Qualified Governmental Tax Information Source or Registration Agency, or from a Qualified Independent Information Source. In addition, the CA MUST validate a Principal Individual associated with the Business Entity pursuant to the requirements in subsection (4), below.
```

Comment:
Should we change "subsection (6)" with "subsection 6." and the various references of (3) (A) (C) to 3. A. C. for consistency?

Reply:
I am open to pull requests, but given that it’s a significant editorial task to do __correctly__,, and that we have zero tooling to enforce it, it seemed rather Pyrrhic. There’s always things we “can” do, but my goal with this ballot was to focus on those that are semantically meaningful for display or the markdown directives itself, or which could be consistently searched and replaced.